aspnetcore/fundamentals/middleware/request-decompression.md

```diff
@@ -69,7 +69,14 @@ In order to guard against [zip bombs or decompression bombs](https://en.wikipedi
 The maximum request size for an endpoint is set by:
 
 * <xref:Microsoft.AspNetCore.Http.Metadata.IRequestSizeLimitMetadata.MaxRequestBodySize?displayProperty=nameWithType>, such as <xref:Microsoft.AspNetCore.Mvc.RequestSizeLimitAttribute> or <xref:Microsoft.AspNetCore.Mvc.DisableRequestSizeLimitAttribute> for MVC endpoints.
```

Comment:
@Rick-Anderson Do you think it would be worth explicitly mentioning here that the limit on `IRequestSizeLimitMetadata` takes precedence over the limit on `IHttpMaxRequestBodySizeFeature`, when the `IRequestSizeLimitMetadata` exists?

Reply:
For sure. I accidently Merged, so can you fix that in another PR.